specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/subscriptions.json

```diff
@@ -0,0 +1,730 @@
+            "description": "Whether to include extended locations.",
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of locations.",
+            "schema": {
+              "$ref": "#/definitions/LocationListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get locations with a subscription id": {
+            "$ref": "./examples/GetLocations.json"
+          },
+          "Get locations with extended locations": {
+            "$ref": "./examples/GetLocationsWithExtendedLocations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}": {
+      "get": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_Get",
+        "description": "Gets details about a specified subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the subscription.",
+            "schema": {
+              "$ref": "#/definitions/Subscription"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a single subscription.": {
+            "$ref": "./examples/GetSubscription.json"
+          }
+        }
+      }
+    },
+    "/subscriptions": {
+      "get": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_List",
+        "description": "Gets all subscriptions for a tenant.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of subscriptions.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get all subscriptions.": {
+            "$ref": "./examples/GetSubscriptions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/tenants": {
+      "get": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "Tenants_List",
+        "description": "Gets the tenants for your account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of tenants.",
+            "schema": {
+              "$ref": "#/definitions/TenantListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
```

Comment:
Not positive why these are not registering with Avocado, but my guess is that it is the period in the example names.  This will need to be fixed before check-in.

Reply:
same for the others.

Reply:
updated.